users/mnil/mnil.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include "keymap_swedish.h"
```

Comment:
If you add the following, you can remove it from the keymap files that include this file.
```suggestion
#include QMK_KEYBOARD_H
#include "keymap_swedish.h"
```